src/scan_modules/unicode_fingerprinting.rs

```diff
@@ -48,7 +50,7 @@ fn surrounding_text(i: usize, text: &str) -> String  {
     surround
 }
 
-impl ScanModule for UnicodeFingerprintScanModule {
+impl ScanModule for UnicodeWatermarkScanModule {
```

Comment:
Is this file redundant to `unicode_watermark.rs`? If so, let's consider deleting it to keep the repository clean.